model/src/u256_decimal.rs

```diff
@@ -0,0 +1,35 @@
+use primitive_types::U256;
+use serde::{de, Deserializer, Serializer};
+use std::fmt;
+
+pub fn serialize<S>(value: &U256, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
Why can we no longer use `serde_with::rust::display_fromstr`? Is it just to get rid of the dependency? 

Ubernit: Should the two serialization mods maybe be combined in a submod (to keep `/src` more easily comprehensible)

Reply:
U256's FromString implementation expects hex but in the api says these are decimal encoded. That's the only reason.